src/marqo/models/create_index_settings.py

```diff
@@ -11,6 +11,8 @@ class IndexSettings(StrictBaseModel):
         type: The type of the index. Can be structured or unstructured.
         allFields: A list of all fields in the index.
         tensorFields: A list of all tensor fields in the index.
+        treatUrlsAndPointersAsImages: Whether to treat urls and pointers as images.
```

Comment:
nit: it could be worth stating in the doc string that this is an unstructured-only param